src/app/_components/userProfile/UserProfile.tsx

```diff
@@ -13,6 +14,20 @@ export function UserProfile(props: UserProfileProps) {
       <h1>Your application is just your profile.</h1>
       <h2>All you have to do is fill it in.</h2>
       {JSON.stringify(props.session, null, 2)}
+
+      <h1>Your team</h1>
+      <p>Team name: {props.team.name}</p>
+      <p>Invite code: {props.team.id}</p>
+
+      <p>Share this QR code with your friends to invite them!</p>
+      <QRCode
+        size={256}
+        style={{ height: "auto", maxWidth: "15%", width: "15%", margin: "2rem"}}
```

Comment:
Any chance we can use Tailwind here? Or the library doesn't let you?

Reply:
Yeppers